tests/Builder/Expression/BsonSizeOperatorTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Expression;
+
+use MongoDB\Builder\Accumulator;
+use MongoDB\Builder\Expression;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+use function MongoDB\object;
+
+/**
+ * Test $bsonSize expression
+ */
+class BsonSizeOperatorTest extends PipelineTestCase
+{
+    public function testReturnCombinedSizeOfAllDocumentsInACollection(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::group(
+                _id: null,
+                combined_object_size: Accumulator::sum(
+                    Expression::bsonSize(
+                        Expression::variable('ROOT'),
```

Comment:
This could be an enum: [PHPLIB-1367](https://jira.mongodb.org/browse/PHPLIB-1367)